docs/maps/connect-to-ems.asciidoc

```diff
@@ -3,17 +3,56 @@
 
 :ems-docker-repo: docker.elastic.co/elastic-maps-service/elastic-maps-server-ubi8
 :ems-docker-image: {ems-docker-repo}:{version}
+:ems-headers-url: https://deployment-host
 
 https://www.elastic.co/elastic-maps-service[Elastic Maps Service (EMS)] is a service that hosts
 tile layers and vector shapes of administrative boundaries.
 If you are using Kibana's out-of-the-box settings, Maps is already configured to use EMS.
 
+[float]
+=== Domains
+
 EMS requests are made to the following domains:
 
 * tiles.maps.elastic.co
```

Comment:
How about changing to the below are something like that to map domain names to the named services in the examples?

* Tile service: tiles.maps.elastic.co
* File service: vector.maps.elastic.co

Reply:
Yeah, I'll improve this thanks